messages/de.json

```diff
@@ -0,0 +1,1393 @@
+        "{0:@} wollte ein neues Leben und entschied, dass dieses Dorf ein guter Ort zum Leben sein würde. Dies erhöhte die Anzahl der Spieler auf  {1:bold}.",
+        "{0:@} kam ins Dorf für einen Urlaub. Dies erhöhte die Spieler Anzahl auf {1:bold}.",
+        "{0:@} überwand die Angst vor Wölfen und beschloss, dem Dorf beizutreten. Dies erhöhte die Spieler Anzahl auf {1:bold}.",
+        "{0:@} ist ein Wolf-Experte, und beschloss so, dem Dorf beizutreten, um sie zu studieren. Dies erhöhte die Spieler Anzahl auf  {1:bold}.",
+        "{0:@} ist besessen von Wölfen, und beschloss das Dorf zu besuchen, um sie zu studieren. Dies erhöhte die Spieler Anzahl auf  {1:bold}.",
+        "{0:@} ist besessen von Lycanthropie, und beschloss das Dorf zu besuchen um zu studieren. Dies erhöhte die Spieler Anzahl auf {1:bold}.",
+        "{0:@} wanderte ins Dorf, während sie nach einem Signal suchten. Glücklicherweise hat das Dorf kostenloses WLAN. Dies erhöhte die Spieler Anzahl auf {1:bold}.",
+        "{0:@} hat im Internet gehört, dass das Dorf ein guter Urlaubsort ist. Dies erhöhte die Spieler Anzahl auf {1:bold}.",
+        "{0:@} hat Gerüchte gehört, dass das Dorf ein guter Urlaubsort ist. Dies erhöhte die Spieler Anzahl auf {1:bold}."
+    ],
+    "bold": "{0:bold}",
+    "fstop_ping": "PING! {0:join_space}",
+    "fool_win": "Spielende. Der {=fool!role} wurde gelyncht dies führte zum Sieg",
+    "no_win": "Spielende! Es gibt keine Spieler mehr.",
+    "piper_win": "Spielende! Jeder ist dem Charme {=piper!role:plural({0})} zum Opfer gefallen . Der {=piper!role:plural({0})} {=führt,führte:plural({0})} die Dorfbewohner weg vom Dorf, nie wieder...kamen diese Zurück",
+    "monster_win": "Spielende! Alle Wölfe sind tot! Als die Dorfbewohner beginnen, den Grill vorzubereiten, Das {=monster!role:plural({0})} {=tötete,tötet:plural({0})} schnell die übrigen Dorfbewohner. Das {=monster!role:plural({0})} hat gewonnen.",
+    "monster_wolf_win": "Spielende! Es gibt die gleiche Anzahl von Wölfen wie unverletzte Dorfbewohner. Die Wölfe überwältigen die Dorfbewohner, werden dann aber von dem {=monster!role:plural({0})} überwältigt, was das {=monster!role:plural({0})} zum sieg führte.",
+    "villager_win": "Spielende! Alle Wölfe sind tot! Die Dorfbewohner hacken sie auf, grillen sie und essen herzhaft.",
+    "wolf_win_equal": "Spielende! Es gibt die gleiche Anzahl von Wölfen wie unverletzte Dorfbewohner. Die Wölfe überwältigen die Dorfbewohner und gewinnen.",
+    "wolf_win_greater": "Spielende! Es gibt mehr Wölfe als unverletzte Dorfbewohner. Die Wölfe überwältigen die Dorfbewohner und gewinnen.",
+    "new_game": "{0:@} hat das Spiel Werwolf gestartet. Schreibe bitte \"{=join!command:!}\" zum Beitreten der Party. Schreibe \"{=start!command:!}\" um abzustimmen das Spiel zu starten. Schreibe \"{=wait!command:!}\" um die wartezeit zu erhöhen.",
+    "you_stasis": "Sorry, aber Sie sind in der Stasi für {0} {=Spiel,Spiele:plural({0})}.",
```

Comment:
"Stasi" ?

Reply:
What is meant by stasis in the game?

Reply:
Warum siezen?

Reply:
mir ist nicht besseres eingefallen tut mir leid :)